### ISSUE TITLE
Graphs Canvas content in Dashboard switches by hovering mouse

### ISSUE THREAD

**Issue by NicolasGoeddel** [home-assistant/frontend#21188]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

I have a dashboard with different graphs showing temperatures and humidity of certain rooms. When I hover over these graphs they switch their content randomly by showing graphs from other canvases on the same dashboard.

Here's a demonstration: https://youtu.be/JuTBmFvTFpQ

### Describe the behavior you expected

I expect it to just show the same graph all the time.

### Steps to reproduce the issue

1. Create dashboard with multiple graphs.
2. Hover mouse over graph and move it.

### What version of Home Assistant Core has the issue?

2024.6.4

### In which browser are you experiencing the issue with?

Firefox 126.0.2 (64-Bit)

### Which operating system are you using to run this browser?

Ubuntu 22.04.4 LTS

### Problem-relevant frontend configuration

```yaml
type: history-graph
entities:
  - entity: sensor.ht_wohnzimmer_temperature
    name: Wohnzimmer °C
  - entity: sensor.ht_wohnzimmer_humidity
    name: Wohnzimmer %
  - entity: sensor.ht_schlafzimmer_temperature
    name: Schlafzimmer °C
  - entity: sensor.ht_schlafzimmer_humidity
    name: Schlafzimmer %
  - entity: sensor.htg3_201_temperature
    name: Nicolas °C
  - entity: sensor.htg3_201_humidity
    name: Nicolas %
  - entity: sensor.htg3_202_temperature
    name: Josi °C
  - entity: sensor.htg3_202_humidity
    name: Josi %
  - entity: sensor.htg3_203_temperature
    name: Dach °C
  - entity: sensor.htg3_203_humidity
    name: Dach %
refresh_interval: 1
title: Thermostate (6h)
hours_to_show: 6
```

### Javascript errors shown in your browser console/inspector

```txt
No errors.
```

**Comment by ildar170975** [comment]
Interesting, failed to reproduce in 2024.6.4 + Chrome/FF + Win10:
![1q](https://github.com/home-assistant/frontend/assets/71872483/61c3941a-9999-44bd-a784-a82c5bf8adb8)
```
      - type: history-graph
        entities: &ref_entities
          - entity: sensor.mijia_300_1_temperature
            name: temp1 °C
          - entity: sensor.xiaomi_cg_1_temperature
            name: temp2 °C
          - entity: sensor.xiaomi_cg_2_temperature
            name: temp3 °C
          - entity: sensor.weather_home_met_temperature
            name: temp4 °C
          - entity: sensor.mijia_300_1_humidity
            name: hum1 %
          - entity: sensor.xiaomi_cg_1_humidity
            name: hum2 %
          - entity: sensor.xiaomi_cg_2_humidity
            name: hum3 %
          - entity: sensor.weather_home_met_humidity
            name: hum4 %
      - type: history-graph
        entities: *ref_entities
        hours_to_show: 2
```

**Comment by NicolasGoeddel** [comment]
Most of the time it is much worse that what you can see on the video. Just moving the mouse a pixel completely changes the content.

It might be an issue of Firefox in combination with Ubuntu and proprietary Nvidia drivers on XServer, but I don't see any issues on any other website I am using. Everything is fine except Home Assistant.

**Comment by NicolasGoeddel** [comment]
In the meantime I found a similar issue with ComfyUI in Firefox.
And I think the reason was that I used a custom [vaapi driver](https://github.com/elFarto/nvidia-vaapi-driver) to get better hardware acceleration for some media types.
I uninstalled everything again and now it seems to work fine.

However graphs are still loading way slower than on my HA Android app. I don't know what kind of issue that could be.

Anyway I will close this now.
